src/containers/WelcomePage/components/LastUsedContent.tsx

```diff
@@ -45,6 +46,11 @@ const LastUsedContent: FC<Props & tType> = ({ articleId, locale, t }) => {
           {article && formatDate(article.updated)})
         </Link>
       )}
+      <EditMarkupLink
+        to={toEditMarkup(articleId, locale)}
```

Comment:
Det er forresten ikkje gitt at artikkelen har en språkversjon som passer med valgt locale, så det beste hadde vore at dette var språkuavhengig. Altså at du kaller toEditMarkup(articleId), men det krever en del endringer for å få til.
Alternativet er å sjekke article.supportedLanguages og bruke locale desom den finnes der, eller bruke det første elementet i lista.

Reply:
I search content så gjøres noe slik:
`toEditMarkup(
    content.id,
    content.supportedLanguages.includes(locale)
      ? locale
      : content.supportedLanguages[0],
  )`
Tenkte jeg kunne kopiere det?